browser/css/jsdialogs.css

```diff
@@ -14,6 +14,11 @@
 	display: flex;
 }
 
+.jsdialog.ui-treeview-body {
+	min-height: 230pt;
```

Comment:
is there any specific reason for using pt here?

a good read https://www.w3.org/Style/Examples/007/units.en.html

Reply:
@pedropintosilva Thanks for this. I modified the file. Also I pulled the latest files from repo yesterday after committing, because of which the commit file shows many changes.